lib/data-fetcher.js

```diff
@@ -33,7 +43,7 @@ const USER_AGENT = ['Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) Appl
 function fetchMetadataDescription(url) {
   return new Promise((resolve, reject) => {
     fetchWithUA(url)
-      .then(response => response.text())
+      .then(response => DOMPurify.sanitize(response.text()))
```

Comment:
Do we need to protect `cheerio` from bogus input? I don't think it's going to do any fetching, etc. itself. Maybe sanitize the description only?